src/main/java/org/springframework/data/convert/CustomConversions.java

```diff
@@ -172,6 +185,36 @@ public void registerConvertersIn(ConverterRegistry conversionService) {
 		VavrCollectionConverters.getConvertersToRegister().forEach(it -> registerConverterIn(it, conversionService));
 	}
 
+	/**
+	 * Delegate check if a {@link PropertyValueConverter} for the given {@literal property} is present via
+	 * {@link PropertyValueConversions}.
+	 *
+	 * @param property must not be {@literal null}.
+	 * @return {@literal true} if a specific {@link PropertyValueConverter} is available.
+	 * @see PropertyValueConversions#hasValueConverter(PersistentProperty)
+	 * @since 2.7
+	 */
+	public boolean hasPropertyValueConverter(PersistentProperty<?> property) {
+		return propertyValueConversions != null ? propertyValueConversions.hasValueConverter(property) : false;
+	}
+
+	/**
+	 * Delegate to obtain the {@link PropertyValueConverter} for the given {@literal property} from
+	 * {@link PropertyValueConversions}.
+	 *
+	 * @param property must not be {@literal null}. param <A> domain specific type
+	 * @param <B> store native type
+	 * @param <C> conversion context type
+	 * @return the suitable {@link PropertyValueConverter} or {@literal null} if none available.
+	 * @see PropertyValueConversions#getValueConverter(PersistentProperty)
+	 * @since 2.7
+	 */
+	@Nullable
+	public <A, B, C extends PersistentProperty<C>, D extends ValueConversionContext<C>> PropertyValueConverter<A, B, D> getPropertyValueConverter(
```

Comment:
Let's make the `getPropertyValueConverter()` method non-nullable as we have `hasPropertyValueConverter(…)` methods to check for the presence of converters.